config/kube-batch-conf.yaml

```diff
@@ -9,3 +9,4 @@ tiers:
   - name: predicates
   - name: proportion
   - name: nodeorder
+enable-backfill: true
```

Comment:
have we addressed my previous comments in this PR?

Reply:
This PR addressed some key issues you raised earlier. The other less urgent issues will be addressed later after this PR gets merged.